packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -199,11 +199,11 @@ The following example outlines how to construct a `RenderQueue`:
 const version = new VersionQuery(stack, 'Version', {
   version: '1.2.3.4',
 });
-const images = new ThinkboxDockerImages(stack, 'Images', {
-  version: version,
-  // Change this to AwsThinkboxEulaAcceptance.USER_ACCEPTS_AWS_THINKBOX_EULA to accept the terms
-  // of the AWS Thinkbox End User License Agreement
-  userAwsThinkboxEulaAcceptance: AwsThinkboxEulaAcceptance.USER_REJECTS_AWS_THINKBOX_EULA,
+const images = new ThinkboxDockerImages(stack, 'Image', {
```

Comment:
Here and others places was changed to `Image` intentionally?

Reply:
Not really intentionally, per se. This was copy/paste from other places. The construct id is pretty immaterial, so it seems fine to me.+